core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1180,6 +1180,13 @@ package object config {
       .intConf
       .createWithDefault(1)
 
+  private[spark] val IO_FILE_UNSPLITTABLE_WARNING_THRESHOLD =
```

Comment:
I just don't think this is worth another config

Reply:
CC @cloud-fan

Reply:
This is also a way to turn off the warning, if people feel it's too annoying. We can make it an internal conf though.